### DISCUSSION TITLE
Could you please add shortcut U and P for “Seek previous/next keyframe” as default

### DISCUSSION THREAD

**Discussion by bytimer** [mifi/lossless-cut#1318] [Ideas]
“Seek previous keyframe”  shortcut add U
“Seek next keyframe” shortcut  add P
**U  I  O  P**
So，press U to previous keyframe and then press I to set "Start current segment at current time" ;
press P to next keyframe and then press O to set "End current segment at current time"
very smoothly，very cool !

**Comment by mifi** [comment]
Not sure why it should be the default, but you can already set up these shortcuts yourself under Settings :)

**Reply by bytimer** [reply]
press U ,and then press I  
press P ,and then press o
very handy！
When on another PC，I need to set the two shortcut key again.. also as "**Reorder segments by start time**"
I prefer to set "Ctrl+S"

**Reply by mifi** [reply]
Im not sure why you are not able to change the shortcuts yourself
